plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/MavenRunnerUtils.kt

```diff
@@ -175,7 +181,7 @@ private fun runMavenInstall(
             transformMavenRunner.run(installParams, mvnSettings, installTransformRunnable)
         } catch (t: Throwable) {
             val error = "Maven Install: Unexpected error when executing bundled Maven install"
-            installTransformRunnable.exitCode(Integer.MIN_VALUE) // to stop looking for the exitCode
+            installTransformRunnable.setExitCode(Integer.MIN_VALUE) // to stop looking for the exitCode
```

Comment:
nit: In the future, avoid bundling refactoring with features. That makes this massive pr harder to read and keep track of than necessary.